### PR TITLE
IN-D Invoice Data Capture

### PR DESCRIPTION
**Pull request by gayathri35**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hello @gayathri35,
Thank you very much and congratulations on your new submission. May I ask you please to register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration). It looks like a new connector, please correct me if I am wrong. 
Please let us know if you have any questions.
Thank you very much again for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @gayathri35,
Thank you so much for registering your new connector. Our Engineers will review your pull request soon.
Please let us know if you have any questions.
Thank you so much again for your hard work with us.

File: certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+                                          "type": "integer",
+                                          "format": "int32",
+                                          "description": "The confidence score for customer gstin extracted from the invoice"
+                                        }
+                                      },
+                                      "description": "customer_gstin"
+                                    }
+                                  }
+                                },
+                                "description": "header_items"
```
Comment (v-EgorKozhadei) [inline certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json:343]: @gayathri35 please look through descriptions for responses, they should look like valid sentences. More information about responses can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#response).

File: certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+            "type": "string",
+            "default": "True",
+            "description": "Accept-Attachment"
+          },
+          {
+            "name": "Accept-Route",
+            "in": "header",
+            "required": true,
+            "type": "integer",
+            "description": "Accept-Route",
```
Comment (v-EgorKozhadei) [inline certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json:54]: @gayathri35 descriptions for parameters should look like valid sentences too. You can refer to [an example from coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @gayathri35!

Please check comments after the review.

Let us know if you have any questions.

File: certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,473 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.1",
+    "title": "IN-D Invoice Data Capture",
+    "description": "A smart invoice processing solution that converts static documents (scanned, PDFs, screenshots, etc.) into usable information to fuel your business processes.IN-D Payables is an AI-led intelligent document classification and capture solution that automates the processing of all kinds of commercial documents used in Shipping, Logistics, and Manufacturing operations. Manual handling of documents slows the pace of business transactions."
+  },
+  "host": "payables-v3.in-d.ai",
```
[the remark this answers is not in the document]
Comment (gayathri35) [reply certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json:8]: Hello @v-EgorKozhadei,
Yes, the specified URL is the production environment.

**Comment by gayathri35** [comment]
Hello @v-EgorKozhadei,
I hope I have cleared the all comment which you sent for IN-D Invoice Capture Connector.Can you please approved and merged the connector.
Thank you so much for your support.

**Comment by v-EgorKozhadei** [comment]
Hello @gayathri35,

Thank you for updating connector files!
I left two comments unresolved from previous review.

Please look through descriptions one more time, there are some grammatical errors and some descriptions don't look like sentences or words (e.g. header_items).

Swagger validation check is failing because x-ms-summary property is missing for operation parameters. Please add this property to operation parameters where it's missing. More information about this property and examples can be found in [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters) and [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).

We'll be good to go after descriptions are fixed and swagger validation passes.

Please let us know if you have any questions or need help.

**Comment by gayathri35** [comment]
Hello @v-EgorKozhadei,
I hope all the comments are resolved, Can you please approve and merged the connector?
Thank you so much for your support.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @gayathri35, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**